docs/opengraph/schema.mdx

```diff
@@ -243,6 +243,59 @@ See Neo4j [Naming and Conventions](https://neo4j.com/docs/cypher-manual/current/
 }
 ```
 
+### Post-processing
+
+Post-processing in BloodHound refers to the analysis phase where the system creates certain edges after ingesting data to identify attack paths.
+
+After ingesting data, BloodHound analyzes the graph state and adds edges it considers useful. BloodHound regenerates "post-processed" edges after it builds a complete graph. Before regenerating post-processed edges, BloodHound deletes any existing ones. As a result, BloodHound removes any post-processed edges that you add directly to an OpenGraph payload.
+
+See the following list of edges that BloodHound creates during post-processing:
+
+- [`AdminTo`](/resources/edges/admin-to)
+- [`AZAddOwner`](/resources/edges/az-add-owner)
+- [`AZMGAddMember`](/resources/edges/az-mg-add-member)
+- [`AZMGAddOwner`](/resources/edges/az-mg-add-owner)
+- [`AZMGAddSecret`](/resources/edges/az-mg-add-secret)
+- [`AZMGGrantAppRoles`](/resources/edges/az-mg-grant-app-roles)
+- [`AZMGGrantRole`](/resources/edges/az-mg-grant-role)
+
+You can work around this behavior by including the supporting edges that cause the post-processing step to generate the edge that you want.
```

Comment:
This assumes that deducing what the supporting edges are is an intuitive process. I'm not sure if that's a realistic assumption.